test/spec/Feature/Query/QuerySpec.hs

```diff
@@ -28,6 +28,19 @@ spec actualPgVersion = do
     it "causes a 404" $
       get "/faketable" `shouldRespondWith` 404
 
+  describe "Sending malformed json" $ do
+    it "causes a 400" $ do
+      post "/items" "}{ x" `shouldRespondWith`
+        [json|{"message":"Empty or invalid json","code":"PGRST102","details":null,"hint":null}|]
+        { matchStatus  = 400
+        , matchHeaders = [matchContentTypeJson]
+        }
```

Comment:
The `QuerySpec.hs` seems to test only `get` requests. I think it's OK to leave it as you did before, both on Insert and Update Specs.

Reply:
Got it, thank you